operations/CSM_product_management/Post_Install_Customizations.md

```diff
@@ -123,7 +175,7 @@ Update resources associated with Prometheus in the `sysmgmt-health` namespace. T
 
 1. Edit the `platform.yaml` to only include the `cray-sysmgmt-health` chart and all its current data. (The resources specified above will be updated in the next step and the version may differ, because this is an example).
```

Comment:
```suggestion
1. Edit the `platform.yaml` to only include the `cray-sysmgmt-health` chart and all its current data. 

   The resources specified above will be updated in the next step. The version may differ, because this is an example.
```